### DISCUSSION TITLE
Video Embedding for recommendation systems

### DISCUSSION THREAD

**Discussion by thomasf1** [milvus-io/milvus#30591] [Ideas & Feature requests]
Is there an implementation for this approach: https://milvus.io/blog/2021-10-10-milvus-helps-analyze-videos.md ?

Would be interesting to compare it to compare it to the performance of pytorchvideo/SlowFast.

Currently, generating embeddings for video seems to be a far way off what is possible for images.

Did anyone else work on this?

**Comment by yhmo** [comment]
An example application is here: https://milvus.io/docs/video_similarity_search.md

**Comment by thomasf1** [comment]
@yhmo Thanks, yhmo - The approach in the examples is different from the article, it´s using timesformer and pytorchvideo via towhee. I wanted to compare the two approaches... :)
